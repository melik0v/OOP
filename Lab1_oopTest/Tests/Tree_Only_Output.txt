It is Tree: age = 25, Name = Pine, Habitat = desert
Consonants = 10
It is Tree: age = 10, Name = Birch, Habitat = desert
Consonants = 20
It is Tree: age = 30, Name = Maple, Habitat = tundra
Consonants = 15

