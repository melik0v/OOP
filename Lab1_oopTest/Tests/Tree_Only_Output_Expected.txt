It is Tree: age = 25, Name = Pine, Habitat = desert
Consonants = 2
It is Tree: age = 10, Name = Birch, Habitat = desert
Consonants = 4
It is Tree: age = 30, Name = Maple, Habitat = tundra
Consonants = 3

